default.nix

```diff
@@ -33,6 +33,7 @@ in
       {
         name ? "${args.pname}-${args.version}",
         verbose ? false,
+        creationTimestamp ? null,
```

Comment:
Could you add a test in the tests directory that fails without this attribute? 

What I would do is add a test that sets `creationTimestamp`, then runs `ripgrep` or `find` to then extract the timestamp from the built file.